modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -0,0 +1,20 @@
+= Connected Apps for Developers and End Users
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Developers
```

Comment:
Linking https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/ here seems critical, since those are the APIs developers will interact with